src/Microsoft.Identity.Client/ITokenCache.cs

```diff
@@ -73,11 +73,12 @@ public interface ITokenCache
         /// otherwise just use <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/>.
         /// </summary>
         /// <param name="msalV3State">Byte stream representation of the cache</param>
+        /// <param name="shouldClearExistingCache">Set to true to clear MSAL cache contents.  Defaults to false.</param>
```

Comment:
Could we give more guidance to developers over when they should set this flag to true / false ?

Reply:
Maybe an aka.ms link to something in the wiki with more details?

---
In reply to: [278059494](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1110#discussion_r278059494) [](ancestors = 278059494)